helm-charts/falcon-sensor/values.yaml

```diff
@@ -2,44 +2,48 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-daemonset:
-  # Annotations to apply to the daemonset
-  annotations: {}
-
-  # additionals labels
-  labels: {}
-
-  updateStrategy: RollingUpdate
-
-  # Various pod security context settings. Bear in mind that many of these have an impact
-  # on the Falcon Sensor working correctly.
-  #
-  # - User that the container will execute as. Typically necessary to run as root (0).
-  # - Runs the Falcon Sensor containers as privileged containers. Typically not
-  #   necessarily unless running within environments such as OpenShift.
-  securityContext:
-    runAsUser: 0
-    privileged: true
-    readOnlyRootFilesystem: false
-    allowPrivilegeEscalation: true
-
-image:
-  repository: falcon-sensor
-  pullPolicy: Always
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "latest"
-
-# Override various naming aspects of this chart
-# Only edit these if you know what you're doing
-nameOverride: ""
-fullnameOverride: ""
-
-podAnnotations: {}
-
-namespace: default
-
-# How long to wait for Filebeat pods to stop gracefully
-terminationGracePeriod: 30
+node:
+  # When enabled, Helm chart deploys the Falcon Senors to Kubernetes nodes
+  enabled: true
+
+  daemonset:
+    # Annotations to apply to the daemonset
+    annotations: {}
+
+    # additionals labels
+    labels: {}
+
+    updateStrategy: RollingUpdate
+
+    # Various pod security context settings. Bear in mind that many of these have an impact
+    # on the Falcon Sensor working correctly.
+    #
+    # - User that the container will execute as. Typically necessary to run as root (0).
+    # - Runs the Falcon Sensor containers as privileged containers. Typically not
+    #   necessarily unless running within environments such as OpenShift.
+    securityContext:
+      runAsUser: 0
+      privileged: true
+      readOnlyRootFilesystem: false
+      allowPrivilegeEscalation: true
+
+  image:
+    repository: falcon-sensor
```

Comment:
Should the images be renamed to ``sensor-node-container`` to match earlier references in the daemonsets.yaml file?

Reply:
Updated them both to be falcon-node-sensor